specification/validated-relationships-service-api.yaml

```diff
@@ -317,57 +322,221 @@ components:
                   type: string
                   description: The patient's NHS number.
                   example: "9000000001"
+        subject: 
+          type: object
+          description: The person who is the subject.
+          properties:
+            type:
+              type: string
+              enum: [Patient, RelatedPerson] # Here: https://simplifier.net/packages/hl7.fhir.r4.core/4.0.1/files/81189/~details this looks like it should be a URI
+            identifier:
+              description: An identifier - identifies some entity uniquely and unambiguously. Typically this is used for business identifiers.
```

Comment:
Think we should make this description specific to our use case and align with the rest of the spec. Suggest:

The patient's NHS number. The primary identifier of a patient, unique within NHS England and Wales. Always 10 digits and must be a [valid NHS number](https://www.datadictionary.nhs.uk/attributes/nhs_number.html).